plugins/net_plugin/net_plugin.cpp

```diff
@@ -2341,14 +2341,6 @@ namespace eosio {
 
    // called from connection strand
    void connection::connect( const std::shared_ptr<tcp::resolver>& resolver, tcp::resolver::results_type endpoints ) {
-      switch ( no_retry ) {
-         case no_reason:
-         case wrong_version:
-         case benign_other:
-            break;
```

Comment:
The `no_retry` check in `resolve_and_connect` has an extra case `duplicate`. Before the removal of the check here, `duplicate` will just `return`. Now `duplicate` will continue. Is this the behavior intended?

Reply:
Yes.